### PR TITLE
Add node counter to the examples

### PR DESCRIPTION
**Pull request by tristanlievens**
Adding a simple node counter, based on the counter example from the redux repo.
Connected to #37

### CODE REVIEW

File: examples/node-counter/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  case 'INCREMENT':
+    return state + 1
+  case 'DECREMENT':
+    return state - 1
+  default:
+    return state
+  }
+}
+
+var store = createStore(counter, devTools())
```
Comment (zalmoxisus) [inline examples/node-counter/index.js]: It should be:
```js
var store = createStore(counter, devTools({ realtime: true }))
```

**Comment by zalmoxisus** [comment]
Thanks for the example. You should either specify `realtime` parameter or have `process.env.NODE_ENV === 'development'`.

**Comment by zalmoxisus** [comment]
I've just [updated the README](https://github.com/zalmoxisus/remote-redux-devtools#important) to specify it more explicitly.

**Comment by zalmoxisus** [comment]
It works with local server, but `ws` throws because of a deprecated parameter. Fixed that in https://github.com/zalmoxisus/remote-redux-devtools/commit/dc31dad22a1a69148d72f0c724fb93778051928b and published `v0.4.2`. Please give it a try.

**Comment by tristanlievens** [comment]
Great! Big thanks 😄

**Comment by zalmoxisus** [comment]
Thank you for the contribution! :-)

**Pull request merged**